tools/check_version_bump.py

```
@@ -169,7 +169,7 @@ def check(base_ref: str | None = None) -> tuple[int, list[str]]:
         code, out, err = _git(*args)
         if code != 0:
             return UNDETERMINED, [
-                "version-bump (ADR-003): could not read the working tree "
+                "version-bump: could not read the working tree "
                 f"({' '.join(args)} failed{': ' + err if err else ''}) — "
                 "refusing to answer from committed history alone"
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)